lib/mapper.js

```diff
@@ -10,19 +10,50 @@ var unixTime = require('unix-time');
 var del = require('obj-case').del;
 var reject = require('reject');
 
+function copyFields(fromObj, toObj, fields) {
```

Comment:
:+1: 

I found this, is it the right library to use/include? https://www.npmjs.com/package/utils-extend